templates/eap-cd-amq-s2i.json

```diff
@@ -17,7 +17,7 @@
     },
     "labels": {
         "template": "eap-cd-amq-s2i",
-        "xpaas": "1.5.0"
+        "xpaas": "14.0"
```

Comment:
IIRC the 'version' field in 'metadata' : 'annotations' above is supposed to match this. 

This same comment applies to all the templates.